Xamarin.Forms.Core.UITests.Shared/BaseTestFixture.cs

```diff
@@ -17,35 +17,10 @@ internal abstract class BaseTestFixture
 
 		public static AppRect ScreenBounds { get; set; }
 
-		[TestFixtureTearDown]
-		protected virtual void FixtureTeardown()
-		{
-		}
-
-		static int s_testsrun;
-		const int ConsecutiveTestLimit = 10;
-
-		// Until we get more of our memory leak issues worked out, restart the app 
-		// after a specified number of tests so we don't get bogged down in GC 
-		// (or booted by jetsam)
-		public void EnsureMemory()
-		{
-			s_testsrun += 1;
-
-			if (s_testsrun >= ConsecutiveTestLimit)
-			{
-				s_testsrun = 0;
-
-				CoreUITestsSetup.LaunchApp();
-
-				FixtureSetup();
-			}
-		}
-
 		[SetUp]
 		protected virtual void TestSetup()
 		{
-			EnsureMemory();
+			FixtureSetup();
```

Comment:
So we're going to restart the app for every single test again? Won't that just increase our UI test run times?

Reply:
First wanted to see if we got this green , and then optimize.